src/logger.js

```diff
@@ -1,14 +1,17 @@
+import color_message from './color'
+
 const LOG_LEVELS = {
   debug: 1,
   info: 2,
   silent: 3
 }
 
-export default function (levelName, message) {
+export default function (levelName, message, color) {
```

Comment:
How about making this a class, and storing options like color in the constructor?

That way we could simply create an instance of the logger and use it without needing to pass `logColor` in each time. 

e.g. 

```javascript
import Logger from 'logger'

logger = new Logger(parsedOptions)
logger(info, 'Test')
```